### PR TITLE
chore: update Fastfile to publish to new distribution groups

### PR DESCRIPTION
**Pull request by scotttwittrockcio**
This PR adds the Firebase Distribution groups for 'feature-branch', 'next', and 'public' in addition to the current 'all-builds' and 'stable-builds' groups.

### CODE REVIEW

**Comment by scotttwittrockcio** [comment]
@Shahroz16 Here is the new PR that used this repo vs the fork as a base. The checks are all passing, and I can confirm it also published to the right groups! Could I get a quick check and approval on this before I merge. I saw the PR template we "Squash and merge". Is there anything else to note before merging after approval?

**Review by Shahroz16 — APPROVED** [review]

**Comment by Shahroz16** [comment]
> @Shahroz16 Here is the new PR that used this repo vs the fork as a base. The checks are all passing, and I can confirm it also published to the right groups! Could I get a quick check and approval on this before I merge. I saw the PR template we "Squash and merge". Is there anything else to note before merging after approval?

@scotttwittrockcio its a chore branch so all you need is `squash and merge` and rest should be good.

**Pull request merged**
